modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -238,12 +252,18 @@ export default class Tileset3D {
     this.refine = json.root.refine;
 
     // TODO add to loader context?
-    this.fetchOptions = this.options.fetchOptions || {};
+    this.loadOptions = this.options.loadOptions || {};
     if (this.options.headers) {
-      this.fetchOptions.headers = this.options.headers;
+      this.loadOptions.fetch = {
+        ...this.loadOptions.fetch,
+        headers: this.options.headers
```

Comment:
Should we make a breaking change and drop the `headers` option as it is now covered by `loadOptions.fetch`? This class is really complex, with so many options. Could certainly be a separate PR.

Reply:
That is fine by me!

Reply:
OK I created an API cleanup section here: #1245 where we can track proposals for API changes.